app/utils.py

```diff
@@ -137,6 +140,12 @@ async def try_dm(account: Account, content: str) -> None:
         print(f"Failed to DM {account} with: {shorten(content, width=50)}")
 
 
+def snowflake_timestamp(snowflake: int) -> dt.datetime:
+    return dt.datetime.fromtimestamp(
+        ((snowflake >> 22) + DISCORD_EPOCH) / 1000, tz=dt.UTC
+    )
```

Comment:
nit: is there a reason why we don't use the official `discord.utils.snowflake_time` instead?

Reply:
Yes: I just missed it 😄 Thanks for the pointer